templates/server.html

```diff
@@ -39,13 +39,13 @@ <h3 class="card-title">Interface</h3>
                             </div>
                             <div class="form-group">
                                 <label for="post_up">Post Up Script</label>
-                                <input type="text" class="form-control" id="post_up" name="post_up"
-                                       placeholder="Post Up Script" value="{{ .serverInterface.PostUp }}">
+                                <textarea class="form-control" id="post_up" name="post_up"
+                                       placeholder="Post Up Script">{{ .serverInterface.PostUp }}</textarea>
```

Comment:
Since `PostUp` and `PostDown` in the generated interface file are single-lined, this change can be a bit tricky because it will allow line-breaks that would probably result in an invalid configuration.